cogs/utils/rsstools.py

```diff
@@ -0,0 +1,75 @@
+""" A collection of functions for RSS feed parsing"""
+
+import discord
+from dateutil import parser
+import regex as re
+from __main__ import botdata
+import aiohttp
+import asyncio
+from bs4 import BeautifulSoup
+
+async def get_html(url):
+    """ Takes url, returns html. No error handling"""
+    async with aiohttp.ClientSession() as session:
+        async with session.get(url) as response:
+            html = await response.text()
+            return html
```

Comment:
Why create a new method for this with no error handling? I'd recommend using `await httpgetter.get(url, return_type="text")`. That way if the http request fails for any reason, the command will fail gracefully

Reply:
Ok. I wrote a new one because I actually missed that http getter could get the text, I was reading through the definitions and saw json everywhere. I will try again with the text option once I move the function to the dota cogs

Reply:
ok gotcha! yeah i definetly should add some documentation for that stuff  because its pretty easy to miss. sounds good.